manifests/otel-agent-cm.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 Google LLC
+# Copyright 2024 Google LLC
```

Comment:
nit: google policy on copyright headers is to set it when the file is created and leave it. So it doesn't need to be updated here.

Reply:
ack